lib/matplotlib/markers.py

```diff
@@ -44,43 +42,44 @@
 CARETLEFTBASE                  caretleft (centered at base)
 CARETRIGHTBASE                 caretright (centered at base)
 CARETUPBASE                    caretup (centered at base)
-"None"                         nothing
-None                           nothing
-" "                            nothing
-""                             nothing
+`" "`, or `""`                 nothing
```

Comment:
This still needs to include the string `'None'` (distinct from the object `None`)